spec/stack.js

```diff
@@ -18,19 +18,52 @@ describe('Stack', () => {
       expect(() => aStack.push('foo'))
         .to.alter(() => aStack.length(), { from: 0, to: 1 })
     })
+    it('pushes an element to the top of the stack.', () => {
+      const aStack = new Stack()
+
+      aStack.push('bar')
+      aStack.push('foo')
+
+      expect(() => aStack[0]
+        .to.equal('foo'))
+    })
+
   })
 
   context('pop()', () => {
     it('removes an element from the top of the stack.', () => {
-      const aStack = new Stack( 1 )
+      const aStack = new Stack(['foo','bar'])
       expect(() => aStack.pop())
-        .to.alter(() => aStack.length(), { from: 1, to: 0 })
+        .to.alter(() => aStack.length(), { from: 2, to: 1 })
+    })
+  })
+
+  context('peek()', () => {
+    it('shows the current element at the top of the stack.', () => {
+      const aStack = new Stack(['foo','bar'])
+      expect(aStack.peek())
+        .to.equal('foo')
```

Comment:
Are you sure that `'foo'` is the thing that should be returned?  Remember, the defining characteristic of a stack is "Last In First Out".  The element returned by `peek()` should be the same element that would be returned by `pop()`, the element at the top of the stack, the one that was most recently pushed.  So...
```
aStack.push('A')
aStack.push('B')
aStack.push('C')

aStack.peek()   // this should return 'C'
```